File: pangram/Pangram.java
import java.util.*;

public class Pangram {
    public boolean checkIfPangram(String sentence) {
        Set<Character> set = new HashSet<>();
        for(int i = 0; i < sentence.length(); i++) {
            char character = sentence.charAt(i);
            if(!set.contains(character) && Character.isLetter(character)){
                set.add(character);
            } 
        }
        return set.size() == 26;
    }
}

File: pangram/Pangram.js
var pangram = function(sentence) {
    let set = new Set();
    for(const character of sentence) {
        if(!set.has(character) && character.match(/[a-zA-Z]/)) {
            set.add(character);
        }
    }
    return set.size === 26;
}
exports.pangram = pangram;